Receptai.client/components/Banner/Banner.vue

```diff
@@ -2,46 +2,140 @@
 import axios from "axios";
 import RecipeBannerComponent from "@/components/Banner/RecipeBannerComponent.vue";
 
-const config = useRuntimeConfig();
-
 interface Recipe {
     id: number;
     name: string;
     shortDescription: string;
+    author: Author;
+    dateCreated: string;
+    dateModified: string | null;
     previewImage: string;
+    tutorialVideo?: string;
+    ingredients: Ingredients[];
+    instructions: string[];
+    tags: string[];
+    categories: Category[];
+    minutesToPrepare: number;
+    portions: number;
+}
+
+interface Author {
+    name: string;
+}
+
+interface Ingredients {
+    purpose: string;
+    ingredients: Ingredient[];
+}
+
+interface Ingredient {
+    name: string;
+    quantity: number;
+    unit: string;
+}
+
+interface Category {
+    id: number;
+    name: string;
+    description: string | null;
+    previewImageUrl: string | null;
+}
+
+const recipeList = ref<Recipe[]>([]);
+const bannerOffset = ref(0);
+const bannerComponentWidth = 192;
+const bannerWidth = ref(0);
+let dragStartX = 0;
+let isDragging = false;
+let moveBannersInterval: NodeJS.Timeout | null = null;
+
+async function fetchRecipes() {
+    try {
+        const response = await axios.get("/api/v1/recipe/list?page=0&sortBy=dateCreated&sortAsc=false");
```

Comment:
You need to use ${config.public.baseURL} before link defined in the axios get, like "${config.public.baseURL}/api/v1/recipe/list?page=0&sortBy=dateCreated&sortAsc=false".
UseRuntimeConfig mus be included to do that.
This is because dev needs to use "localhost:3000" while prod used "localhost"